src/search.cpp

```diff
@@ -1161,14 +1161,18 @@ namespace {
           if (thisThread->ttHitAverage > 509 * TtHitAverageResolution * TtHitAverageWindow / 1024)
               r--;
 
-          // Reduction if other threads are searching this position
+          // Increase reduction if other threads are searching this position
           if (th.marked())
               r++;
 
           // Decrease reduction if position is or has been on the PV (~10 Elo)
           if (ss->ttPv)
               r -= 2;
 
+          // Increase reduction at root and non-PV nodes when the best move does not change frequently
+          if (!(PvNode && !rootNode) && depth > 10 && thisThread->bestMoveChanges <= 2)
```

Comment:
To me
`if ((rootNode || !PvNode) && depth > 10 && thisThread->bestMoveChanges <= 2)`
looks much more easy to read/understand.

Reply:
it also corresponds 1:1 to what the comment says